src/Language/PureScript/Errors.hs

```diff
@@ -1195,6 +1197,15 @@ renderBox = unlines
   dropWhileEnd p = reverse . dropWhile p . reverse
   whiteSpace = all isSpace
 
+evalTypelevelFunctions :: Type -> Type
```

Comment:
Right, I interpreted it more as a general function like "Just evaluate whatever you can on the type level!". I think we can still keep this general function, but then in the actual error message handling I'll check if after evaluation, I end up with something of kind Symbol and then display it, otherwise skip to the next case.